SConstruct

```diff
@@ -81,7 +81,7 @@ for x in sorted(glob.glob("platform/*")):
         continue
     tmppath = "./" + x
 
-    sys.path.insert(0, tmppath)
+    sys.path.insert(1, tmppath)
```

Comment:
CC @touilleMan if you have any suggestion.

Reply:
I agree with @ajreckof, changing the insert order risks causing name clash issues. It's already a hack changing the path order the way we do, if we start messing with it because other things rely on some random thing being in a certain place in the path array, it's only going to be worse.

Scons provides objects that resolve paths to the root - `Dir("#").abspath` or `File("#nameofdile").abspath`. If a script doesn't have access to these Scons functions, than having the path to the root available from an env var, like @ajreckof suggested, would be a great fallback.

Reply:
Thanks both for the advice! Both options sound good. I don't know why I didn't think of `Dir("#")` since we're in SConstruct and it's available... I tested it and it works, so we don't need to hack `sys.path` further.